azure-pipelines.yml

```diff
@@ -94,19 +94,20 @@ stages:
         ALL_IT_TESTS=$(Whitelist_Bedrock.ALL_IT_TESTS)
         AZ_COMMON_KV_TEST=$(Whitelist_Bedrock.AZ_COMMON_KV_TEST)
         mkdir -p $(Pipeline.Workspace)/variables
+        echo "Whitelisting Integration Test Values 1. ALL_TESTS: $ALL_IT_TESTS 2. AZ_SIMPLE: $AZ_SIMPLE_TEST 3. AZ_COMMON: $AZ_COMMON_KV_TEST 4. AZ_MC: $AZ_COMMON_MC_TEST"
         echo "$ALL_IT_TESTS" > $(Pipeline.Workspace)/variables/ALL_IT_TESTS
         echo "$AZ_SIMPLE_TEST" > $(Pipeline.Workspace)/variables/AZ_SIMPLE_TEST
         echo "$AZ_COMMON_KV_TEST" > $(Pipeline.Workspace)/variables/AZ_COMMON_KV_TEST
         echo "$AZ_COMMON_MC_TEST" > $(Pipeline.Workspace)/variables/AZ_COMMON_MC_TEST
       displayName: Exporting Integration Test Trigger Variables to Pipeline Workspace
     - publish: $(Pipeline.Workspace)/variables
       artifact: variables
-
-- stage: Bedrock_Integration_Test_Validation
-  jobs:
+#- stage: Bedrock_Integration_Test_Validation
```

Comment:
@NathanielRose did you mean to leave this commented and not deleted?

Reply:
Comments Removed